src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -540,6 +540,7 @@ def load_arguments(self, _):
         c.argument('display_name', arg_type=ts_display_name_type)
         c.argument('description', arg_type=ts_description_type)
         c.argument('version_description', arg_type=ts_version_description_type)
+        c.argument('prompt', options_list=['--yes', '-y'], action='store_true', help='Do not prompt for confirmation')
```

Comment:
May I ask if `--yes` is passed in for this parameter, then there will be no prompt, right? If so, is it better to change the parameter name to `no_prompt`? Because the meaning of this name is more clear and accurate.